config/300-function-manager.yaml

```diff
@@ -14,3 +14,12 @@ spec:
             image: github.com/vmware/dispatch/cmd/dispatch-server
             args:
             - function-manager
+            - --image-registry berndtj
+            - --sourceroot /store
+            volumeMounts:
```

Comment:
I thought this didn't work.

Reply:
Yeah, it doesn't :).  I'm reverting all my changes to config, but like I said, I'd like to delete it